python/ray/autoscaler/batching_node_provider.py

```diff
@@ -191,6 +203,7 @@ def node_tags(self, node_id: str) -> Dict[str, str]:
             TAG_RAY_NODE_KIND: node_data.kind,
```

Comment:
Only add `TAG_RAY_REPLICA_INDEX` when `node_data.replica_index` is not None.

Reply:
Done in 47e46ac.